transport/messages/result_message_base.hh

```diff
@@ -42,6 +50,9 @@ public:
     class prepared;
     class schema_change;
     class rows;
+
+private:
+    std::vector<sstring> _warnings;
```

Comment:
We usually put fields at the top of the class.

Reply:
The implementation comes before the interface? All right... Will change.

Reply:
> I was in the middle of reviewing when a force-push removed my last comment, so I'll just post it here:
> 
> Will old CQL-v3 clients understand the message if it contains a `warning` flag and a list of strings?The flag itself will be ignored as https://github.com/apache/cassandra/blob/trunk/doc/native_protocol_v3.spec says, but what about the list of strings?
> Please verify.
That's why I did the force push :D